crates/pallet-subspace/src/lib.rs

```diff
@@ -390,6 +397,7 @@ mod pallet {
     impl<T: Config> GenesisBuild<T> for GenesisConfig {
         fn build(&self) {
             SegmentIndex::<T>::put(0);
+            NextSalt::<T>::put(1u64.to_le_bytes());
```

Comment:
In some places, we use little-endian bytes, in some other places we use big-endian bytes, just curious about the reason for this choice.

Reply:
little-endian is what most platforms use, so that is what we tend to use lately for tiny efficiency gain. We do use big-endian for tag search though as in that case we get a nice property of being able to use RocksDB's range search to effectively treat keys as numbers and find solutions within 64-bit solution range. In this particular case it doesn't matter too much as long as it is the same on all nodes, hence little-endian default is used.